packages/blockchain-wallet-v4-frontend/src/data/components/taxCenter/selectors.ts

```diff
@@ -0,0 +1,56 @@
+import { FIRST_YEAR } from 'blockchain-wallet-v4-frontend/src/scenes/TaxCenter/utils'
+
+import { Types } from '@core'
+import { selectors } from 'data'
+
+import { getData as getDataBch } from '../../../scenes/Settings/Addresses/Bch/Wallets/selectors'
+import { getData as getDataXlm } from '../../../scenes/Settings/Addresses/Xlm/selectors'
```

Comment:
I don't think we want to be traversing the directory structure like this. It can quickly lead to circular dependencies. Those selectors aren't too complex so I would just copy the selectors into this file instead of requiring them here.

Reply:
removed